deploy/helm/kubecf/templates/ingress.yaml

```diff
@@ -99,36 +100,71 @@ spec:
               servicePort: 443
 {{- else }}
 ---
+{{- with $service := index .Values.services "router" }}
```

Comment:
I don't quite understand why this isn't just `with $service := .Values.services.router`…

Reply:
I first wanted to make it a template and loop over the values.yaml. But that didn't work. This is just a left-over of that. Same for the below comment.

Reply:
Right, so we should probably actually remove the loop attempt.